src/readsbml.jl

```diff
@@ -659,7 +659,7 @@ function get_model(mdl::VPtr)::SBML.Model
                     Cint,
```

Comment:
If we're ***sure*** this is a boolean value, this can also be
```suggestion
                    Bool,
```
and you don't need to add `!= 0` below, with the caveat that `1` is `true`, and all the rest is `false`:
```julia
julia> f = @cfunction identity Cint (Cint,)
Ptr{Nothing} @0x00007fa9afb184d0

julia> ccall(f, Cint, (Cint,), 2)
2

julia> ccall(f, Bool, (Cint,), 2)
false

julia> ccall(f, Bool, (Cint,), 1)
true

julia> ccall(f, Bool, (Cint,), 0)
false
```
which might be a bit counter-intuitive.

Reply:
```julia
julia> ccall(f, Bool, (Cint,), 6)
false
```

ok well now this is scary. I'd use it as-is because that is what we have in the get_optional_bool above, and also the function signatures should match.